src/createServer.js

```diff
@@ -1,3 +1,37 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+const { handleErrors } = require('./handleErrors');
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    res.setHeader('Content-type', 'application/json');
+
+    const normalizedURL = new URL(req.url, `http://${req.headers.host}`);
+    const textToConvert = normalizedURL.pathname.slice(1);
+    const toCase = normalizedURL.searchParams.get('toCase');
+
+    const errors = handleErrors(textToConvert, toCase);
+
+    if (!errors.errors.length) {
+      const convertedText = convertToCase(textToConvert, toCase);
+
+      req.statusMessage = 'OK';
```

Comment:
I think, that statusMessage for statusCode = 200 is Ok by default.

Reply:
yeah, OK should by default